src/core/os/amdgpu/amdgpuScreen.cpp

```diff
@@ -131,21 +131,19 @@ const SwizzledFormat PresentableSwizzledFormat[2] =
     {
         ChNumFormat::X8Y8Z8W8_Srgb,
         { ChannelSwizzle::Z, ChannelSwizzle::Y, ChannelSwizzle::X, ChannelSwizzle::W }
-    }
-};
-
-const SwizzledFormat PresentableHdrSwizzledFormat[2] =
```

Comment:
If we're rolling these formats into the PresentableSwizzledFormat array, then I think it makes sense to remove the (now empty) PresentableHdrSwizzledFormat array and simplify the appropriate logic inside Screen::GetFormats.

Reply:
Ok, thanks for the review. I will make these changes and update commit in this pull request.

Reply:
Done, and successfully retested.